### ISSUE TITLE
Unable to connect with ssl set to false

### ISSUE THREAD

**Issue by JBraungardt** [jmerriweather/exldap#10]
After upgrading to 0.6.1 I'm not able to connect to a server with ssl set to false.
It seems that is not valid to sent sslopts when ssl is set to false.

After changing
```
def open(server, port, ssl, timeout, sslopts) do
    :eldap.open([to_charlist(server)], [{:port, port}, {:ssl, ssl}, {:sslopts, sslopts}, {:timeout, timeout}])
end
```

to
```
def open(server, port, ssl, timeout, sslopts) do
    :eldap.open([to_charlist(server)], [{:port, port}, {:ssl, ssl}, {:timeout, timeout}])
end
```
the connection was successful.

**Comment by jmerriweather** [comment]
Thanks for reporting the issue, should now be fixed in 0.6.2. I've also added a test to make sure this doesn't slip through next time.

**Comment by martin-langhoff** [comment]
Ah, looks like I broke it. Apologies!
